homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -80,6 +80,8 @@ def __init__(self, device, bridge):
         self._device_type = device["type"]
         self._device_name = device["name"]
         self._device_zone = device["zone"]
+        self._device_model = device["model"]
```

Comment:
This is getting out of control, just store `device` 😉

Reply:
`_state` was actually a reference to `device`, but set at a later time. I've combined them.